docs/quickstarts/nextjs.mdx

```diff
@@ -46,38 +48,40 @@ pnpm add @clerk/nextjs
 
 </CodeBlockTabs>
 
-### Set environment keys
+### Set your environment keys
 
-In your Next.js project's root folder, you may have an `.env.local` file alongside `package.json` and other configuration files. If you don't see it, create it.
+1. If you don't have a `.env.local` file in the root directory of your Next.js project, create one now.
+2. Find your secret keys in your Clerk Dashboard:
+    - Navigate to your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys) to find your secret keys.
+    - Select your application, then select **🔑 API Keys** in the sidebar menu.
+    - You can copy your keys from the **Quick Copy** section.
+3. Add your keys to your `.env.local` file. The final result should look as follows:
 
-Add the following code to your `.env.local` file to set your public and secret keys.
+    <InjectKeys>
 
-**Pro tip!** If you are signed into your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys), your secret key should become visible by clicking on the eye icon.
+    ```sh filename=".env.local"
+    NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+    CLERK_SECRET_KEY={{secret}}
+    ```
 
-<InjectKeys>
+    </InjectKeys>
 
-```sh filename=".env.local"
-NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
-CLERK_SECRET_KEY={{secret}}
-```
+<Callout type="info">
 
-</InjectKeys>
+If you're signed into Clerk, the above snippet will contain your secret key. Select the eye icon to reveal it. Otherwise, find your API keys in [the Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys).
 
-This sets your public and secret keys.
+</Callout>

```

Comment:
I think this can get moved to be above the "Inject keys" part. and it won't need to say "Otherwise, find your API keys..." any more because with your new additions, there are clear instructions of how to find them.

Reply:
Good point 👍 I've added this line to the instructions above the code snippet instead